baybe/constraints/conditions.py

```diff
@@ -61,6 +81,13 @@ class Condition(ABC, SerialMixin):
     Conditions are part of constraints, a constraint can have multiple conditions.
     """
 
+    def generate_operator_function(self):
```

Comment:
I refactored a part of code into this method according to Adrian's comment [here](https://github.com/emdgroup/baybe/pull/252#discussion_r1623900562).
You're right, since it's only used in one subclass (`ThresholdCondition`) I move it to that specific subclass.